src/renderer/components/Editor/WordDragManager.tsx

```diff
@@ -123,15 +123,22 @@ const WordDragManager = ({ clearSelection, children }: Props) => {
       }
 
       if (event.altKey) {
-        // If alt/option held, then start a drag-move action
+        // If alt/option held, then start a drag-move action.
+
+        // Pauses (where word text === null) cannot be dragged around
+        const isPause = (words ?? [])[wordIndex].word === null;
+        if (isPause) {
+          return;
```

Comment:
Following on from Irmy and Justin's suggestion to be able to copy/paste pauses, should we also be able to drag pauses around if the pause is like an action or movement? But I guess that might depend on what the UI's gonna look like for pauses.

Reply:
Solid chance we'll be nuking drag as multi drag isn't supported (and would be a lot of work) and it's not performance optimised